highlight.io/components/Competitors/sentry.ts

```diff
@@ -0,0 +1,157 @@
+import datadogsmall from '../../public/images/datadog-small.png'
+import datadogfull from '../../public/images/datadog.png'
+import { Competitor } from './competitors'
+import {
+	HighlightIsOpenAndTransparent,
+	HighlightShipsNewFeatures,
+} from './shared-snippets'
+
+export const SentrySpec: Competitor = {
+	name: 'Sentry',
+	header: 'A Truly Open Source Sentry Alternative.',
+	subheader:
+		'Session replay of your frontend, fullstack error monitoring, and powerful logging.',
+	logoDesktop: datadogfull,
```

Comment:
Looks like we need to s/datadog/sentry/g in this file.

Reply:
thanks. added the logos.